frontend/src/app/main/main.component.html

```diff
@@ -0,0 +1,3 @@
+<md-input [type]="'text'" placeholder="Input"></md-input>
```

Comment:
I am not really sure. They don't say wether you need it or not. [md-input](https://github.com/angular/material2/blob/master/src/lib/input/README.md)